values/wire-server/prod-secrets.example.yaml

```diff
@@ -67,3 +67,95 @@ team-settings:
+    b+u3apymOkCMBAXSIMD1jk90R4vfvEzLT/i8K2AfbtxYDL3TTBTC/SREkOKEf15/
+    GP2Q1p+ItNmDDDS559CCJ0UceQ14skfj62RNmAcTJyy4JL4kc9+6CsTgJApzuR+f
+    wDDlvtyx47ZBc6nywXYcJORRoI9GMEXHLzTza+0CggEALxV1UCKoNyaD80Zxtw3D
+    jTKZb8gYWaBH+kPmgVMaJ9AI22ddGT8wq871L0g3vuf/pm55xQkFnF03j7u0ufQG
+    wCyUPDBXo8BElTxN7qW3KeC15ViFvBE1ejrdUr6p0cH5gT1ElNF0UgYnYeEpyp1e
+    VjAxzaSon4LNp94Va/ZjMbpfYXuBVYJpl/XKRh9kRGpTMG0wRKQMSCyXrtB3BaWa
+    uTXe+j84zZPc3ydn2EXbncMGBozHyXxnOXMt0xUQSWWE2LbGYf3hHLZQ9IPt2eyT
+    Zop//ywRi/6Mq3HvNKMG3kUUzazPXR7JRUIAomEuK8PDw/Dgu4zUD0krgkb8tsAT
+    /QKCAQA9xb1J1WOmjf7izklyy9C/z9tU4TiG+yQb0CuPAvJg3psvt6YxhgJ5K8EQ
+    LwItt9eS+G8t0IZRVr+tcW0LNNVwhiZ6jB+U8EOtMsEhvSu0HgE80N077VDKNGby
+    vxMaEcaLdV55lvB0BMuQz3nEjKtG4xxsStWLGsgtFOemworWELwEaSTq3J9O4TdB
+    zL1R7XANDg2/aSJ1FOvTXB23z+MZ3KJ5vMTT/Ab2ObZlIsoh3aqvCXmglm1zu9b4
+    jnVykMzTxNvEziManB9ImPUsKUm/xC7Z6JMoqaC18GsoL1lVcESxxDPojBE9yS97
+    8XoOMfTNdIlwz95nELMF+xraLgzA
+    -----END PRIVATE KEY-----
+
+  tlsCrt: |
+    -----BEGIN CERTIFICATE-----
+    MIIFITCCAwmgAwIBAgIUU0zwEIpV7xgSJqeLhtDQ4N2ojF4wDQYJKoZIhvcNAQEL
+    BQAwIDEeMBwGA1UEAwwVbGVnYWxob2xkLmV4YW1wbGUuY29tMB4XDTIwMTExMzEy
+    MDQwNFoXDTIxMTExMzEyMDQwNFowIDEeMBwGA1UEAwwVbGVnYWxob2xkLmV4YW1w
+    bGUuY29tMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAp7ae9SlGcwJA
+    jC2hB5AavWMpu+4e1w0lyfRISoxeX8In1sPhjPNZIo3Sr/4BV8zduEpQi+hg43JP
+    MPHMH+It18IXVrJEVuOfIIfmOJh0ZrdRO66ukAtEwEsz5u+Jk1kZrPI5TV9W3ZFf
+    XuEFKNyAr7UE1Bp45LajPV8kSOxxe6oCnpjj5+s1ZCSKPWUbJ4l1wT291cH14EVC
+    GkFDunVNV7fC7uigqsAuz98ECAERAV7ES/vJyBPo5jEknOV9lcBZDCjPHcb1TMOi
+    fViIZ9JrpJV1u6REzqgFTB/Wf+zDS6RXxIkaLZXxdz20amUDb6Nl+xMc3twET6GK
+    AN4Hn1SnbM1wWENgtEu+MhaOusZdc8Vjw/QV2HhoK0uytxQdlQXBCZ+25Zzh1qCe
+    fT5Tcy0b3YSovHeiEWzTPNZ336H+H9RILE70Q97Rncpr6vdlyo2GWhNnQrWOKMyd
+    xauhdjX7vncXFFSMu5ro2p0nib7DvFemk5QDE2ovlrdf+GBMJukypo+aEQlbKQhp
+    vspSDbdVJU94A67s6LsczaONwX3ursJAjn39XLcZJxafLdRfjpjs3S9a0NFf12mD
+    hJcZbVQiuoUwfECu0ax6b1ia+9EU/uVnjthLkz44N5Kk9MfIiu3eQDOfKaWQr5D8
+    awukq8Q73dKHwOFesO3qhXIpSSM/pLcCAwEAAaNTMFEwHQYDVR0OBBYEFKSX+vVF
+    pxu6vyeU9aV5V7OabMTPMB8GA1UdIwQYMBaAFKSX+vVFpxu6vyeU9aV5V7OabMTP
+    MA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggIBAJ+FSuc7Xej1odVe
+    EuNWzZvUfbLl43yrXjIZK1KwxclC2AkQ55iI/8FBkd9eXWIjTOM+nxZJFSrjE+2k
+    lDekVhn5+VCIqL4BbXFJorfQLz0tkujcIWhQ463EZTTQhKQ/X6B/rGM74X/NkSsK
+    uGva6vpOYZIgrIYYi0okhpe8hu8mA3HwvotGCaRQoUTssrX9J62pvuRkUPKeuO4Q
+    Fx5uTMc2SHluXKtQQiqkyX5BLzYIspjd7SxDT33w2gaqptIhIl7Ip5E7XLCZ8Ro4
+    gbm7DBFcPqWoGzORMWLCDVqAFLwSBNnr9gKnCUyVrnqr0VvyWME2nVFRKZaNoX65
+    D2ypqFmOJsCLYnvHGlUoUTpzRMYEejMUvCQJK8Z9FFjI3TWh8u7qei5JhBb6BDY8
+    YPkHG4rG3DK6G4nWc4o60ig3sXoXvYct18xQhaLvGlVqvD1TZHctkHPUc3pm9MOU
+    eyIZ+E3V2fBu7ej+F6zCCHkbe6QjfyH5EHyhSWJ40fa/zj+HjV49Y73XMqVv3jIC
+    MV+1P9Fasxs1+f2fo0A1zx0ZPH2rCuZnkab0WdScmORXzZiDJfXQ9eY1M2k92s3h
+    Y6OuY6Uvi8fN54myQv5tU7Ko/nzG3yNIEW74UULB4Kh/Zb5FaBCIlRZiJosivkns
+    AjSlxx2XPfQbuybjYmkc28tp7gWA
+    -----END CERTIFICATE-----
```

Comment:
suggestion: for consistency, you may want to follow [this existing approach](https://github.com/wireapp/wire-server-deploy/blob/develop/values/nginx-ingress-services/prod-secrets.example.yaml) for placeholders.

Reply:
Hmm but having placeholders that work is kind of nice.   I have double feelings about this. I agree that we should do it consistently though.  I'll remove them for now

Reply:
> having placeholders that work is kind of nice.

Good point. But would legalhold really work with that cert and key? Another perspective could be that it might actually be desired to fail as soon as possible (maybe even during `helm install`), if both are not valid. Though, I'm not sure if that the existing approach would cause such behavior.

Reply:
So if they're not _set_ then the chart gives a useful error message at the moment (using the `required` filter in the templates) . How about I comment the sections out? So that people get a nice error message telling them they should not forget to set this value?

The certificate here is the certificate for `legalhold.example.com`.  Indeed note that useful. I'll remove the snakeoil certs and comment out the sections; so that people are clearly instructed they should set this value if they enable legalhold.

Sound good?